src/content/docs/accounts/original-accounts-billing/original-users-roles/user-migration.mdx

```diff
@@ -30,44 +30,46 @@ Requirements include:
+* If you require account-level roles for user management capabilities. Roles related to user management (ability to add and update users, change user type, create access grants) currently apply across an entire organization and can't be assigned to specific accounts. 
 
 If you have questions about whether you should use the user migration wizard, talk to your New Relic account representative. 
 
-## Understand user management concepts [#before-beginning]
+## Optional: review your users' user type [#user-type]
+
+For organizations on the [New Relic One pricing model](/docs/accounts/accounts-billing/new-relic-one-pricing-billing/new-relic-one-pricing-billing), your users' [user type](/docs/accounts/accounts-billing/new-relic-one-user-management/user-type) is a billing factor. 
+
+Whether you're currently on the New Relic One pricing model, or plan to soon [switch to that pricing model](/docs/accounts/original-accounts-billing/original-product-based-pricing/switch-new-models), it may make sense to edit your users' user type before doing the user migration. One reason for that is that the original **Users and roles** UI allows you to see the time of your users' last use of New Relic (the new UI does not yet have this ability), and that can be useful for determining which user type to make them. For tips on how to do this, see [Edit user type](/docs/accounts/original-accounts-billing/original-users-roles/users-roles-original-user-model/#full-user-count).
+
+## Optional: Understand user management concepts [#before-beginning]
 
 During the user migration procedure, you'll have a choice to either grant all your users access to all your organization's accounts, or to assign more granular access (set the roles and accounts your user groups have access to). 
 
 The more you need to partition access to accounts or roles, the more it will help you to understand some basics about the newer user management concepts. Here's a brief summary: 
 
-* Users are grouped in a container called an "authentication domain". The domain governs how users are added to New Relic (manually with the UI or automatically via SCIM). It also governs how users log in (manually with email/password or using SAML SSO). Most organizations will have just one or two authentication domains: one for the default manual settings and another for the more automatic methods.
+* Users are in a container called an "authentication domain". The domain governs how users are added to New Relic (manually with the UI or automatically via SCIM). It also governs how users log in (manually with email/password or using SAML SSO). Most organizations will have just one or two authentication domains: one for the default manual settings and another for the more automatic methods.
```

Comment:
I'd add a comma: One for the default manual settings, and another one for the more automatic methods